### PR TITLE
Ics2 3255 - Changes for adding a new service in the list

### PR DESCRIPTION
**Pull request by jmsatheesh**

### CODE REVIEW

File: app/uk/gov/hmrc/testuser/models/TestUser.scala [hunk 1]
```diff
@@ -123,12 +124,13 @@ case class TestOrganisation(
     pensionSchemeAdministratorIdentifier: Option[String] = None,
     eoriNumber: Option[String] = None,
     groupIdentifier: Option[String] = None,
+    personPresentingTheGoods: Option[String] = None,
     override val services: Seq[ServiceKey] = Seq.empty,
     override val _id: BSONObjectID = BSONObjectID.generate,
     crn: Option[String] = None,
-    taxpayerType: Option[String] = None
+    taxpayerType: Option[String] = None,
```
Comment (peteslater-ee) [inline app/uk/gov/hmrc/testuser/models/TestUser.scala]: Remove trailing comma

File: app/uk/gov/hmrc/testuser/models/TestUser.scala [hunk 2]
```diff
@@ -123,12 +124,13 @@ case class TestOrganisation(
...
   ) extends TestUser {
-  override val affinityGroup = "Organisation"
+  override val affinityGroup = "Organisation";
```
Comment (peteslater-ee) [inline app/uk/gov/hmrc/testuser/models/TestUser.scala]: Remove semi-colon

File: app/uk/gov/hmrc/testuser/models/TestUser.scala [hunk 3]
```diff
@@ -317,7 +322,8 @@ case class TestOrganisationCreatedResponse(
     override val eoriNumber: Option[String] = None,
     override val groupIdentifier: Option[String] = None,
     override val crn: Option[String] = None,
-    override val taxpayerType: Option[String] = None
+    override val taxpayerType: Option[String] = None,
+    override val personPresentingTheGoods: Option[String] = None,
```
Comment (peteslater-ee) [inline app/uk/gov/hmrc/testuser/models/TestUser.scala]: Remove trailing comma

File: app/uk/gov/hmrc/testuser/models/TestUser.scala (hunk 2 above)
Comment (jmsatheesh) [reply app/uk/gov/hmrc/testuser/models/TestUser.scala]: Removed semi-colon

File: app/uk/gov/hmrc/testuser/models/TestUser.scala (hunk 1 above)
Comment (jmsatheesh) [reply app/uk/gov/hmrc/testuser/models/TestUser.scala]: Removed trailing comma

File: app/uk/gov/hmrc/testuser/models/TestUser.scala (hunk 3 above)
Comment (jmsatheesh) [reply app/uk/gov/hmrc/testuser/models/TestUser.scala]: Removed trailing comma

**Comment by chriswaugh** [comment]
We're going to persue an alternate approach for ICS. This PR can be closed.

**Comment by AndySpaven** [comment]
As requested, closing this now defunct PR

**Pull request closed without merging**